project/build.properties

```diff
@@ -1 +1 @@
-sbt.version=1.3.3
+sbt.version=1.4.7
```

Comment:
Maybe let's just go back to `1.3.3` here and `ci.yml`? Let's see if it works then.